test/embulk/input/zendesk/test_plugin.rb

```diff
@@ -586,6 +586,36 @@ def run_task
             end
           end
 
+          sub_test_case ".config_to_task" do
+            data do
+              [
+                ['http:/example.com', 'http:/example.com'],
+                ['htt://example.com', 'htt://example.com'],
+                ['http://.example.com', 'http://.example.com'],
+                ['://example.com', '://example.com'],
+                ['http://example.', 'http://example.'],
```

Comment:
Currently, non-HTTPS is acceptable, so if we want to reject non-HTTPS, I think we should have another PR.